src/ert/trace.py

```diff
@@ -0,0 +1,11 @@
+from opentelemetry import trace
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import SpanLimits, TracerProvider
+
+resource = Resource(attributes={SERVICE_NAME: "ert"})
+tracer_provider = TracerProvider(
+    resource=resource, span_limits=SpanLimits(max_events=128 * 16)
```

Comment:
I suspect this number is much higher than what we really need, but let's leave it like this for the time being.
Events within a span are meant for noticeable changes like exceptions, outcomes and such.